types/interfaces.ts

```diff
@@ -182,3 +182,6 @@ export enum UploadModes {
   NewModel = 'newModel',
   NewVersion = 'newVersion',
 }
+
+export enum DateStringBrand {}
+export type DateString = string & DateStringBrand
```

Comment:
Is it worth adding a comment explaining that `DateString` is used to represent date strings in ISO 8601 format?